docs/conf.py

```diff
@@ -61,39 +62,39 @@
 # You can specify multiple suffix as a list of string:
 #
 # source_suffix = ['.rst', '.md']
-source_suffix = '.rst'
+source_suffix = ".rst"
 
 # The master toctree document.
-master_doc = 'index'
+master_doc = "index"
 
 # General information about the project.
-project = u'Cortex Python Library'
-copyright = u'2020, CognitiveScale'
-author = u'CognitiveScale'
+project = "Cortex Python Library"
+copyright = "2020, CognitiveScale"
```

Comment:
2023

Reply:
yeah changed all years in all python files to 2023. We can do the Copyright owner change to TecnoTree in another PR?